packages/resilient/abbr/init.lua

```diff
@@ -39,6 +39,7 @@ function package:registerCommands ()
 
   self:registerCommand("abbr:no", function (_, content)
     local lang = SILE.settings:get("document.language")
+    lang = lang:sub(1, 2) -- We only need the language code, so truncate any BCP-47 tag
```

Comment:
Not all our non-BCP-47 language codes are two letters, some are 3; e.g. `und`. I would discard everything after separators like `[_-.]`.

Reply:
@alerque Oops, right indeed!